src/rules/noFlowFixMeInStrictFiles.js

```diff
@@ -0,0 +1,43 @@
+const FLOW_STRICT_MATCHER = /^\s*@(?:no)?flow\s*strict(?:-local)?\s*$/u;
+
+const isStrictFlowFile = (context) => {
+  return context
+    .getAllComments()
+    .some((comment) => {
+      return FLOW_STRICT_MATCHER.test(comment.value);
+    });
+};
+
+const message =
+  'No suppression comments are allowed in "strict" Flow files. Either remove the error supression, or lower the strictness of this module.';
+
+const create = (context) => {
+  if (!isStrictFlowFile(context)) {
+    // Skip this file - nothing to check here
+    return {};
+  }
+
+  return {
+    Program: () => {
+      const comments = context
+        .getSourceCode()
+        .getAllComments()
+        .filter((node) => {
+          return node.type === 'Block' || node.type === 'Line';
+        });
+
+      for (const comment of comments) {
+        if (/\$FlowFixMe/iu.test(comment.value)) {
```

Comment:
just like I created in a different rule, you may want to consider other suppression error names like `FlowExpectedError` which is my preference
ref: https://github.com/gajus/eslint-plugin-flowtype/pull/489/files#diff-ca0fd280a4ec7ba166e4acdb000cdad6a3f6a000d71ad8632113895e58774260R13

Reply:
Happy to! Are you suggestion I copy/paste this definition in here too?

```js
const suppressionTypes = ['$FlowFixMe', '$FlowExpectedError', '$FlowIssue', '$FlowIgnore'];
```

Or is there a central "utility file" I could import this definition from?

Reply:
I think just copy and paste for now. Since my pr isn't merged either, room for some utility constant in the future though

Reply:
OK, done in af3ee31d4198c03b3bc288570c68071d7ef7dbd2 — lmk what you think 👍